docker/Dockerfile.alpine-tracee-make

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3.15 as alpine-tracee-make
```

Comment:
I'm afraid the directory name `docker` might be confusing to some, as it will signal that this is where the dockerfile/artifact scripts are. maybe rename it to `docker-builder` or something? (not sure, if you think it's clear than leave it)

Reply:
Yep. mentioned this in the other comment. Let's call it "builder" and have everything related to building (but the default Makefile) there.